analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/DisablingRequestValidationBase.cs

```diff
@@ -83,7 +83,7 @@ protected override void Initialize(SonarAnalysisContext context)
                     return;
                 }
 
-                foreach (var fullPath in context.ProjectConfiguration(c.Options).FilesToAnalyze.FindFiles("web.config"))
+                foreach (var fullPath in context.ProjectConfiguration(c.Options).FilesToAnalyze.FindFiles("web.config").Where(File.Exists))
```

Comment:
How is it possible that we get some files from the scanner that do not exist?

Reply:
it's a good question, and it's also good to be defensive.
But it's a good question that we need to ask.

Reply:
If you mess up your project file or targets file, you can explicitly include a file that's not on the disk, but will be in the MsBuild list of files. From there, it gets propagated to FilesToAnalyze and we have fun with those.